### DISCUSSION TITLE
Black Font in the SQL Editor by the Fusion Dark Style

### DISCUSSION THREAD

**Discussion by lukas-adrian** [pawelsalawa/sqlitestudio#4655] [General]
### Details
I am using Fusion Dark as Style. If I open the "SQL Editor" the Font is black and you can't see cleary.
But just open the Configuration Form and the Font will be white (you can see it in the background). After restart is black again.

### Steps to reproduce
- Change the Style to Fusion Dark.
- Open SQL Editor and type something, the font is directly black
- (maybe restart)

### Operating system
Windows 10, all updates

### SQLiteStudio version
3.4.1

**Comment by pawelsalawa** [comment]
I'm not sure about the reason. Maybe you have some kind of a conflict in the configuration with some older config values. Please try going to `C:\Users\googie\AppData\Local\SalSoft\SQLiteStudio\` and rename the `settings3` file to some other name. Do it while SQLiteStudio is NOT running. Then start SQLiteStudio and apply style and see if that works.

Yes, it will require for you to configure everything again (if this workaround works). Unfortunately.

**Comment by lukas-adrian** [comment]
Thanks Pawel, its working!

Pozdrawiam

Lukasz
